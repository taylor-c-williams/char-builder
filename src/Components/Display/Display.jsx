export default function Display({ catchphrases }) {
	return (
		<div>
			Catchphrases:
			{catchphrases.map((catchphrase) => (
				<p key={catchphrase}>"{catchphrase}"</p>
			))}
		</div>
	);
}
