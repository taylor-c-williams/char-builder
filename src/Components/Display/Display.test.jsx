import { screen, render } from '@testing-library/react';
import Display from './Display';

it('Should render Picker component', () => {
	const { container } = render(
		<Display catchphrases={['bingo!', 'jabroney!']} />
	);

	expect(container).toMatchSnapshot();
});
