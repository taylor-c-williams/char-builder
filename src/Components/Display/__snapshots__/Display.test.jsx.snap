// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Picker component 1`] = `
<div>
  <div>
    Catchphrases:
    <p>
      "
      bingo!
      "
    </p>
    <p>
      "
      jabroney!
      "
    </p>
  </div>
</div>
`;
